static/css/event-meta.less

```diff
@@ -0,0 +1,65 @@
+/* ==========================================================================
+   cfgov-refresh
+   event-meta
+   ========================================================================== */
+
+/* topdoc
+  name: Event Meta
+  family: cfgov-event-meta
+  notes:
+    - "Meta information to display the event location, date, and time"
+  patterns:
+    - name: Event Meta
+      markup: |
+        <section class="event-meta h-event">
+          <div class="event-meta_address h-adr">
+            <span class="event-meta_city p-locality">San Francisco</span>,
+            <span class="event-meta_state p-region">CA</span>
+            <span class="event-meta_venue p-extended-address">Super Dome</span>
+          </div>
+          <time class="dt-start" datetime="2014-01-21 2:30 PM CST">
+            <span class="event-meta_date">1/21/2014</span>
+            <span class="event-meta_time">2:30 PM CST</span>
+          </time>
+        </section>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          section.event-meta.h-event
+            div.event-meta_address.h-adr
+              span.event-meta_city.p-locality
+              span.event-meta_region.p-region
+              span.event-meta_venue.p-extended-address
+            time
+              div.event-meta_date
+              div.event-meta_time
+  tags:
+    - cfgov-event-meta
+*/
+
+.event-meta {
+  .h5();
```

Comment:
This mixin adds a `text-transform: uppercase` but the mockup shows title case for the city and venue.